### PR TITLE
updated gitignore and added migrations

### LINKED ISSUE #948: updating .gitignore and adding migrations folder to all the the apps
## Description
As a developer,
I need to update the .gitignore file and add the migrations folder within the apps
so that I can apply migrations to the apps and add files .vscode in gitignore so that this folder can be ignored while staging changes.

## Mocks
image

### Add
- [ ] migrations folder to all the apps

### Update 
- [ ] gitignore

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
10 minutes

### PR DESCRIPTION
**Pull request by hardikkat24**
# Description
VSCode and Sublime Text files added to .gitignore and added migrations to all apps.

Fixes #948 

# Type of Change:
**Delete irrelevant options.**

- Quality Assurance

**Code/Quality Assurance Only**

# How Has This Been Tested?
I made sure that Sublime Text files were not uploaded and verified that migrations folder were present in apps.

# Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -103,3 +103,17 @@ local/
 # Documentation build
 _build
 
+# Visual Studio Code
```
Comment (yashshah2820) [inline .gitignore:106]: Just include the .vscode folder

File: .gitignore [hunk 2]
```diff
@@ -103,3 +103,17 @@ local/
...
 
+# Visual Studio Code 
+.vscode/* 
+!.vscode/settings.json 
+!.vscode/tasks.json 
+!.vscode/launch.json 
+!.vscode/extensions.json 
+.history
+
+# Sublime Text
```
Comment (yashshah2820) [inline .gitignore:114]: add the folder instead of individual files

File: vms/event/migrations/0001_initial.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
```
Comment (yashshah2820) [inline vms/event/migrations/0001_initial.py:1]: remove this file

File: vms/job/migrations/0001_initial.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
```
Comment (yashshah2820) [inline vms/job/migrations/0001_initial.py:1]: remove this migrations files you just need to add the migration folder

**Comment by hardikkat24** [comment]
I have done the changes. Please review and tell if there are any more changes.

**Comment by yashshah2820** [comment]
@anitab-org/qa-team this PR is ready to be merged

**Comment by hardikkat24** [comment]
@yashshah2820 Thanks!!

**Comment by rpattath** [comment]
> remove the migrations files and .model.py.swp, you just need to add migration folder with only **init**.py inside it

@yashshah2820 @hardikkat24 I verified the changes in the above comment. Are there any functionality testing required?

**Comment by yashshah2820** [comment]
> > remove the migrations files and .model.py.swp, you just need to add migration folder with only **init**.py inside it
> 
> @yashshah2820 @hardikkat24 I verified the changes in the above comment. Are there any functionality testing required?

I have reviewed the PR no other testing is required.

**Comment by hardikkat24** [comment]
> > remove the migrations files and .model.py.swp, you just need to add migration folder with only **init**.py inside it
> 
> @yashshah2820 @hardikkat24 I verified the changes in the above comment. Are there any functionality testing required?

Yes, no further testing is required.

**Pull request merged**